### PR TITLE
add frontend skeleton

### LINKED ISSUE #9: FE: Frontend skeleton
Messaging Centre 

- Left sidebar
- Main Content

### PR DESCRIPTION
**Pull request by hanuz06**
add frontend skeleton

### CODE REVIEW

File: client/src/containers/MainContentField/style.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+export const useStyles = makeStyles((theme) => ({
+  root: {
+    // margin: theme.spacing(2),
```
Comment (bonnieli) [inline client/src/containers/MainContentField/style.js]: make sure you clean up this commented out code if not unused before merging :)

File: client/src/containers/MainPage/index.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React, { useEffect, useState } from "react";
+import { Box } from "@material-ui/core";
```
Comment (bonnieli) [inline client/src/containers/MainPage/index.jsx]: remove some of these imported modules if unused

File: client/src/containers/MainContentField/style.js (hunk 1 above)
Comment (hanuz06) [reply client/src/containers/MainContentField/style.js]: Removed commented out code

File: client/src/containers/MainPage/index.jsx (hunk 2 above)
Comment (hanuz06) [reply client/src/containers/MainPage/index.jsx]: Removed unused imported modules

**Comment by olzhas-kalikhan** [comment]
I reviewed code I think it's fine to merge with main branch

**Pull request merged**